src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.Configuration/ConfigUpdaterSubStep.cs

```diff
@@ -27,7 +27,18 @@ public ConfigUpdaterSubStep(UpgradeStep parentStep, IConfigUpdater configUpdater
             _configUpdater = configUpdater ?? throw new ArgumentNullException(nameof(configUpdater));
         }
 
-        protected override bool IsApplicableImpl(IUpgradeContext context) => context?.CurrentProject is not null && (_parentStep?.ConfigFiles.Any() ?? false);
```

Comment:
We no longer need the ConfigFiles piece of this conditional?

Reply:
The parent step (ConfigUpdaterStep) checks that there are config files and won't apply if there aren't any, so we don't need to check it here in the sub-steps also. I removed this because it was causing issues because I moved around when IsApplicable is called a bit and now the ConfigFiles aren't available when this is invoked.